app/controllers/holds_controller.rb

```diff
@@ -35,6 +37,60 @@ def new
     @bib = Bib.new(parsed_body, holdable_locations)
   end
 
+  # Handles placing holds
+  #
+  # POST /holds
+  def create
+    barcodes = params['barcode'].present? ? Array(params['barcode']) : params['barcodes']
+
+    raise HoldCreateException, 'Error' if barcodes.blank?
```

Comment:
Thinking one improvement would be to have a consistent plural `barcodes` for #create to expect to simplify this code and eliminate the extra check on whether or not a value is there (present? and blank?).

```rb
    raise HoldCreateException, 'Error' if params['barcodes'].blank?
    
    barcodes = [params['barcodes']].flatten
```

This would require changing `place_hold_hidden_input_component.html.erb` to

```
<%= hidden_field_tag :barcodes, bib.barcode %>
```

Reply:
Yes, this would be great, I will add these changes in 👍

Reply:
This refactor is definitely about making hold work with new which were developed separately in parallel so there's bound to be stuff like this 👍

Reply:
:100:

Reply:
btw just committed the related changes 👍